sql/errors.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Liquidata, Inc.
```

Comment:
I like this, we had too much in `core.go`

Reply:
:D